(WIP) exclude/skip 9k-Ex

chrisvanheuveln opened the pull request:
Mike and I decided to close this PR and make the changes post-release since there are 40+ files to change with this. (see commit [74019bb](https://github.com/cisco/cisco-network-node-utils/pull/647/commits/74019bbb86a06d21fdfc011650829823aefe873b))

chrisvanheuveln committed:
exclude/skip 9k-Ex

mikewiebe commented on lib/cisco_node_utils/cmd_ref/stp_global.yaml:
```diff
@@ -7,7 +7,7 @@ _template:
 
 bd_designated_priority:
-  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
+  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k-EX, N9k]
```
Are you planning to add `N9k-EX` as a recognized platform?

chrisvanheuveln replied:
yes

chrisvanheuveln committed:
Add N9k-EX to list of allowed platforms

- Note: The following files will also need to be updated: (basically need to change every place that `[..., N9k, N9k-F, ...]` exist)

```
lib/cisco_node_utils/cmd_ref/snmp_community.yaml:  N9k-F: *template_structured
lib/cisco_node_utils/cmd_ref/snmp_community.yaml:  N9k-F: *acl_structured_get
lib/cisco_node_utils/cmd_ref/snmp_community.yaml:  N9k-F: *all_structured
lib/cisco_node_utils/cmd_ref/snmp_community.yaml:  N9k-F: *group_structured_get
lib/cisco_node_utils/cmd_ref/README_YAML.md:include the various Nexus switch product lines (`N3k`, `N3k-F`, `N5k`, `N6k`. `N7k`, `N9k`, `N9k-F`). When using one or more product keys in this fashion, you can also use the special key `else` to handle all other products not specifically called out:
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  os_version: 'N9k-F:7.0.3.F2.1'
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N5k, N6k, N7k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  os_version: 'N9k-F:7.0.3.F2.1'
lib/cisco_node_utils/cmd_ref/route_map.yaml:  N9k-F: *set_extcommunity_device_str
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N5k, N6k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N5k, N6k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  os_version: 'N35:7.0.3.I7.1; N9k-F:7.0.3.F2.1; N9k:7.0.3.I5.1'
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N5k, N6k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N5k, N6k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/route_map.yaml:  os_version: 'N35:7.0.3.I7.1; N9k-F:7.0.3.F2.1; N9k:7.0.3.I5.1'
lib/cisco_node_utils/cmd_ref/route_map.yaml:  _exclude: [N35, N3k, N5k, N6k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/ip_multicast.yaml:_exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/feature.yaml:  _exclude: [N3k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/feature.yaml:  _exclude: [N3k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/feature.yaml:  _exclude: [C3048, C3064, C3132, N5k, N6k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/feature.yaml:  _exclude: [N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/feature.yaml:  _exclude: [N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/feature.yaml:  _exclude: [N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/itd_service.yaml:_exclude: [N3k, N5k, N6k, N3k-F, N9k-F, ios_xr]
lib/cisco_node_utils/cmd_ref/bgp_neighbor.yaml:  _exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/evpn_stormcontrol.yaml:_exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/show_version.yaml:  N9k-F: *template3k
lib/cisco_node_utils/cmd_ref/bgp.yaml:  os_version: 'N7k:8.0; N3k, N9k:7.0.3.I5.1; N9k-F:7.0.3.F3.2'
lib/cisco_node_utils/cmd_ref/bgp.yaml:  os_version: 'N7k:8.0; N3k, N9k:7.0.3.I5.1; N9k-F:7.0.3.F3.2'
lib/cisco_node_utils/cmd_ref/bgp.yaml:  _exclude: [ios_xr, N3k, N3k-F, N7k, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/interface_evpn_multisite.yaml:_exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/fabricpath_topology.yaml:_exclude: [N3k, N3k-F, N9k-F, N9k, ios_xr]
lib/cisco_node_utils/cmd_ref/vdc.yaml:_exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k, ios_xr]
lib/cisco_node_utils/cmd_ref/vrf.yaml:  N9k-F: *vni9k
lib/cisco_node_utils/cmd_ref/itd_device_group.yaml:_exclude: [N3k, N5k, N6k, N3k-F, N9k-F, ios_xr]
lib/cisco_node_utils/cmd_ref/acl.yaml:  _exclude: [N5k, N6k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/snmp_server.yaml:  N9k-F: *n3k_default_packet_size
lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml:  _exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml:  N9k-F: *soft_recon_always
lib/cisco_node_utils/cmd_ref/bfd_global.yaml:  _exclude: [N3k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/bfd_global.yaml:  _exclude: [N3k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/bfd_global.yaml:  _exclude: [N3k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/bfd_global.yaml:  os_version: 'N9k-F:7.0.3.F2.1; N9k:7.0.3.I6.1'
lib/cisco_node_utils/cmd_ref/evpn_multicast.yaml:_exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/vtp.yaml:_exclude: [ios_xr, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface_portchannel.yaml:  N9k-F: *lacp_max_bundle_32
lib/cisco_node_utils/cmd_ref/aaa_authorization_service.yaml:  N9k-F: *remove_local_allowed
lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml:  _exclude: [N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml:  os_version: 'N9k, N9k-F:9.2'
lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml:  _exclude: [N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml:  os_version: 'N9k, N9k-F:9.2'
lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml:  N9k-F: *mt_lite_default
lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml:    _exclude: [N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/portchannel_global.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/portchannel_global.yaml:  N9k-F: &bundle_hash_ip_l4port
lib/cisco_node_utils/cmd_ref/portchannel_global.yaml:  _exclude: [N3k, N5k, N6k, N7k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/portchannel_global.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/portchannel_global.yaml:  _exclude: [N3k, N7k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/portchannel_global.yaml:  N9k-F: &load_balance_type_no_hash
lib/cisco_node_utils/cmd_ref/portchannel_global.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/portchannel_global.yaml:  _exclude: [N5k, N6k, N7k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/portchannel_global.yaml:  _exclude: [N5k, N6k, N7k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/vlan.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vlan.yaml:  N9k-F: *mapped_vni_n3_8_9k
lib/cisco_node_utils/cmd_ref/vlan.yaml:  _exclude: [N3k, N3k-F, N9k, N9k-F]
lib/cisco_node_utils/cmd_ref/vlan.yaml:  _exclude: [N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/vlan.yaml:  _exclude: [N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/bridge_domain.yaml:_exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k, ios_xr]
lib/cisco_node_utils/cmd_ref/encapsulation.yaml:_exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k, ios_xr]
lib/cisco_node_utils/cmd_ref/vrf_af.yaml:  _exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/vrf_af.yaml:  _exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/vrf_af.yaml:  _exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/dhcp_relay_global.yaml:  N9k-F: *relay_default_true
lib/cisco_node_utils/cmd_ref/dhcp_relay_global.yaml:  _exclude: [N3k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/dhcp_relay_global.yaml:  _exclude: [N7k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/dhcp_relay_global.yaml:  _exclude: [N5k, N6k, N7k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/dhcp_relay_global.yaml:  N9k-F: *v6_relay_default_true
lib/cisco_node_utils/cmd_ref/ospf.yaml:  _exclude: [ios_xr, N3k, N7k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [ios_xr, N3k, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/interface.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vpc.yaml:_exclude: [ios_xr, N3k-F, N9k-F]
lib/cisco_node_utils/cmd_ref/vpc.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vpc.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vpc.yaml:  _exclude: [N3k, N5k, N6k, N7k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vpc.yaml:  _exclude: [N3k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vpc.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vpc.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vpc.yaml:  _exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vpc.yaml:  _exclude: [N3k, N5k, N6k, N7k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/bgp_af.yaml:  _exclude: [ios_xr, N3k, N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/vxlan_vtep_vni.yaml:  _exclude: [N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/vxlan_vtep_vni.yaml:  _exclude: [N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/vxlan_vtep_vni.yaml:  _exclude: [N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/vxlan_vtep_vni.yaml:  os_version: 'N9k, N9k-F:9.2'
lib/cisco_node_utils/cmd_ref/vxlan_vtep_vni.yaml:  _exclude: [N3k-F, N9k-F, N9k]
lib/cisco_node_utils/cmd_ref/interface_service_vni.yaml:_exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k, ios_xr]
lib/cisco_node_utils/cmd_ref/evpn_multisite.yaml:_exclude: [ios_xr, N3k, N3k-F, N5k, N6k, N7k, N9k-F]
lib/cisco_node_utils/cmd_ref/fabricpath.yaml:_exclude: [N3k, N3k-F, N9k-F, N9k, ios_xr]
lib/cisco_node_utils/cmd_ref/bridge_domain_vni.yaml:_exclude: [N3k, N5k, N6k, N3k-F, N9k-F, N9k, ios_xr]
```

chrisvanheuveln committed:
Lint

chrisvanheuveln commented:
This change is too risky this close to release so we'll park it for now and re-visit post-release.
